requirements.txt

```diff
@@ -2,6 +2,7 @@ cftime
 pywps==4.4
 xclim==0.24.0
 xarray
+xesmf>=0.5.3
```

Comment:
Houston, on Pypi https://pypi.org/project/xesmf/#history xesmf is only 0.3.0 !

Reply:
Yup. We're using https://github.com/pangeo-data/xESMF

Reply:
The core developer vanished, so we forked the repo and maintain it elsewhere.

Reply:
My point is that line will never work if xesmf on Pypi is stuck at 0.3.0.

Reply:
To be clear, xESMF cannot be installed with pip. It has to be installed with conda. 
However, once it's installed, my understanding is that we can still ask setup.py to check that the installed version meets some requirements, irrespective of how it was installed.